README.md

```diff
@@ -116,17 +116,7 @@ For more information on how to use Pyroscope with other programming languages, i
 
 You can download the latest version of pyroscope for macOS, linux and Docker from our [Releases page](https://github.com/grafana/pyroscope/releases).
 
-## Supported Languages
-
-* [x] Go (via `pprof`)
-* [x] Python (via `py-spy`)
-* [x] Ruby (via `rbspy`)
-* [x] Linux eBPF
-* [x] Java (via `async-profiler`)
-* [x] Rust (via `pprof-rs`)
-* [x] .NET
-* [x] PHP (via `phpspy`)
-* [x] Node
+## [Supported Languages](https://grafana.com/docs/pyroscope/latest/configure-client/)
```

Comment:
I was in two minds if this link would even be better:

https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/#available-profiling-types

Reply:
We could do both. We could have a subsection under this header that links to support profiles types.